src/runtime/tests/arc-test.ts

```diff
@@ -1040,9 +1040,10 @@ describe('Arc', () => {
     recipe.normalize();
     await arc.instantiate(recipe);
 
-    const rootSlotConsumer = slotComposer.consumers.find(c => !c.arc.isInnerArc) as HeadlessSlotDomConsumer;
-    await rootSlotConsumer.contentAvailable;
-    assert.strictEqual(rootSlotConsumer._content.template, 'ABCDEFGHIJKLMNOPQRSTUVWXYZ');
+    // TODO(sjmiles): content no longer captured this way
```

Comment:
If I have my way SlotConsumer itself will be gone by the end of the transition, so tests of this form will be gone. If we want to restore testing of a certain behavior, it will need to be built against slotObserver machinery.